src/Database/Connection.php

```diff
@@ -121,6 +122,19 @@ public function __construct($config)
     }
 
     /**
+     * Destructor
+     *
+     * Disconnects the driver to release the connection.
+     */
+    public function __destruct()
+    {
+        if ($this->_transactionStarted && class_exists('Cake\Log\Log')) {
+            Log::warning('The connection is going to be closed but the transaction is still active.');
+        }
+        unset($this->_driver);
```

Comment:
I think it would be better to explain more detail about what had happened.
@motooka We have removed this `unset()` call in #11049. However, you have reverted the change in your merge commit accidentally. I cannot see any new changes in your merge commit. Could you please *merge*, not overwrite.
Also, I would suggest to use `git rebase` instead of `git merge` to solve conflicts. See [this help](https://book.cakephp.org/3.0/en/contributing/code.html#submitting-a-pull-request) for contributing code.